lib/openxml/elements/run_properties.rb

```diff
@@ -32,9 +32,12 @@ class RunProperties < OpenXml::Container
         words
       ]
 
-      def initialize(*boolean_properties, **value_properties)
+      def initialize(*boolean_properties, font_color: nil, typeface: nil, **value_properties)
         super()
 
+        self.font_color = font_color
+        self.typeface = typeface
+
```

Comment:
Wouldn't the `public_send` in the `value_properties` loop assign these just as well? Any reason to call them out with explicit assignments?